### ISSUE TITLE
Order By Multiple Fields

### ISSUE THREAD

**Issue by justinrsmith** [JoshAshby/pyRethinkORM#7]
Currently it seems that I can only orderBy on one field? If I do something like 

``` python
Records.orderBy('artist')
```

but if I want to do something like 

``` python
Records.orderBy('artist', 'album')
```

I do not receive a proper sort. In SQLAlchemy I know you can do something like

``` python
Records.orderBy(Record.artist, Record.album)
```

and if you want.

``` python
Records.orderBy(Record.artist.asc(), Record.album.asc())
```

Just wondering what your thoughts are on adding some functionality like this for sorting? I could be completely missing something too and be able to sort by multiple fields and just being blind to the fact. I'd be more than happy to give it a look.

**Comment by JoshAshby** [comment]
You are correct that you can only do orderBy on only one field right now, but there isn't anything that your missing, just me being lazy and not having a need at the moment with my own use of this to order on multiple fields. This is definitely something that could easily happen; since most of collections is just a convenience wrapper around the driver rather than some fancy-dancy super class, it should be as simple as:

``` python
def order_by(self, key1, *args):
    self._query = self._query.order_by(key1, *args)
    return self._query
```

So that it'd be easy to also say which direction you want, just like the regular ReQL driver (`r.desc('row_name')` and stuff).

While I do like the looks of something like `Records.orderBy(Record.artist.asc(), Record.album.asc())` I don't know how possible (well, possible yes but how clean I should say) that'll be since collections and models don't have any knowledge about the structure of the data (since its free to be different per document).

I'd feel fine about getting this worked into dev, but since its something that would probably be considered a breaking change, I'd rather not have it made on the current pre-v1.0.0 releases.

**Comment by justinrsmith** [comment]
Ok ya doing it your way there worked no problem so I may just use that for now since that suits my needs pretty much perfectly. If I get some time I may play with the dev branch and see if I can come up with something, it may be a bit "hack-ish" though, so we will see.
